### ISSUE TITLE
Which user closed a task ?

### ISSUE THREAD

**Issue by bluegroper** [kanboard/forum#47]
We're happily using KB version 1.0.30.
Sometimes we'd like to know which user has closed a task.
Does a later version of KB track this action by user ?
TIA's

**Comment by creecros** [comment]
I don't believe it's a stored property, but do know task close is an event and that you could send an email or notification upon close which would tell you who closed it.
